.dockerignore

```diff
@@ -2,7 +2,6 @@
 .dockerignore
 .git
 .travis.yml
-Dockerfile
```

Comment:
Docker complained that the Dockerfile was ignored when I tried to build the image. Not sure how it has worked so far if this is a build-breaker...